apps/meteor/app/lib/server/functions/updateMessage.ts

```diff
@@ -12,7 +12,7 @@ import { validateCustomMessageFields } from '../lib/validateCustomMessageFields'
 import { parseUrlsInMessage } from './parseUrlsInMessage';
 
 export const updateMessage = async function (
-	message: AtLeast<IMessage, '_id' | 'rid' | 'msg'>,
+	message: AtLeast<IMessage, '_id' | 'rid' | 'msg' | 'customFields'>,
```

Comment:
Why do we need to have `customFields` here being required?

Reply:
![image](https://github.com/user-attachments/assets/09430791-3930-4114-beb8-c92efe1223d0)

Custom fields should retain the `optional` part as atleast just "picks" the types from the base one.